cloud/azure/variables.tf

```diff
@@ -23,7 +23,25 @@ variable "poll_rate" {
 variable "services" {
   description = "Azure service metrics to import. Empty list imports all services"
   type        = list
-  default     = []
+
+  # 2020-05-11 - List from https://www.terraform.io/docs/providers/signalfx/r/azure_integration.html#service-names
+  default = [
```

Comment:
I'd rather see https://github.com/terraform-providers/terraform-provider-signalfx/issues/202 addressed instead of having to manually keep this list updated in the module.

Reply:
Yep, agreed, I opened the issue right after this PR :) (which is now in Draft)

Reply:
Done, using data_source released in https://github.com/terraform-providers/terraform-provider-signalfx/pull/204 & v4.20.0